mava/systems/ippo/networks.py

```diff
@@ -187,6 +188,8 @@ def make_discrete_networks(
     policy_layer_sizes: Sequence[int],
     critic_layer_sizes: Sequence[int],
     observation_network: Callable = utils.batch_concat,
+    orthogonal_initialisation: bool = False,
+    activation_function: Callable[[jnp.ndarray], jnp.ndarray] = jax.nn.relu,
```

Comment:
Probably it will be better to mention that these attributes are Optional? 
`orthogonal_initialisation: Optional bool = False,` ...

Reply:
At the moment `orthogonal_initialisation` is not an optional parameter but it rather getting set to `False` by default. So I am not sure whether adding `Optional` to the type hint will work?